schemas/com.snowplowanalytics.snowplow.ecommerce/promotion/jsonschema/1-0-0

```diff
@@ -0,0 +1,62 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a promotion entity in Ecommerce",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow.ecommerce",
+        "name": "promotion",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "id": {
+            "description" : "The ID of the promotion.",
+            "type": "string",
+            "maxLength": 4096
+        },
+        "name": {
+            "description" : "The name of the promotion.",
+            "type": [ "string", "null" ],
+            "maxLength": 256
+        },
+        "products" : {
+            "description" : "Array of SKUs or product IDs showcased in the promotion.",
+            "type" : [ "array", "null" ],
+            "items" : {
+                "type" : "string",
+                "maxLength": 4096
+            }
+        },
+        "position": {
+            "description" : "The position the promotion was presented in a list of promotions E.g. banner, slider.",
+            "type" : [
+                "integer",
+                "null"
+            ],
+            "minimum": 0,
+            "maximum": 9999999
+        },
+        "creative_id": {
+            "description" : "Identifier/Name/Url for the creative presented on the promotion.",
+            "type" : [
+                "string",
+                "null"
+            ],
+            "maxLength": 256
```

Comment:
If the field should accept URLs as well (as stated in description), 256 max length seems too short.